### PR TITLE
[RSS-24] Announcement Card Component 

### PR DESCRIPTION
**Pull request by esauerbo**
Screen Shot 2021-01-27 at 2 22 20 PM

Created announcement card component.

New one with bootstrap:
Screen Shot 2021-01-30 at 12 08 42 PM

### CODE REVIEW

**Review by ellabuysse — COMMENTED** [review]
Looks good!

File: src/components/AnnouncementCard/AnnouncementCard.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    font-weight: lighter;
+    letter-spacing: 0.5px;
+  }
+  .arrow {
+    border: solid #6666FF;
+    border-width: 0 3px 3px 0;
+    padding: 5px;
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
+  }
```
Comment (jackfales) [inline src/components/AnnouncementCard/AnnouncementCard.css]: Can you delete this file? I don't think it's doing anything because you used Bootstrap

**Review by jackfales — CHANGES_REQUESTED** [review]
Looks good! Just delete the .css file I commented on and re-request my review and I'll approve it.

**Review by jackfales — APPROVED** [review]
Looks good! Great work.

**Pull request merged**